examples/src/main/java/com/flipkart/gjex/examples/helloworld/web/HelloWorldHealthCheckResource.java

```diff
@@ -0,0 +1,35 @@
+package com.flipkart.gjex.examples.helloworld.web;
+
+import com.flipkart.gjex.core.health.ManualRotationManagement;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Singleton
+@Path("/app-healthcheck")
+@Named
+public class HelloWorldHealthCheckResource {
+  private ManualRotationManagement manualRotationManagement;
+
+  @Inject
+  public HelloWorldHealthCheckResource(ManualRotationManagement manualRotationManagement) {
+    this.manualRotationManagement = manualRotationManagement;
+  }
+
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/")
+  public Response health() {
+    if (manualRotationManagement.inHealthy()) {
+       return Response.status(Response.Status.OK).build();
+    } else {
+      return Response.status(Response.Status.SERVICE_UNAVAILABLE).build();
+    }
+  }
+}
```

Comment:
Why is this resource required? Wouldn't the `HealthCheckResource` itself provide this?

Reply:
This is not required because this functionality is handled by HealthCheckResource. Deleted.